apps/docs/integrations/aws-ses.mdx

```diff
@@ -45,10 +45,10 @@ export default function Email(props) {
 
 Import the email template you just built, convert into a HTML string, and use the AWS SES SDK to send it.
 
-```js index.tsx
+```js index.ts
 import { render } from '@react-email/render';
 import AWS from 'aws-sdk';
-import { Email } from './email';
+import Email from './email';
```

Comment:
Why the change here?

Reply:
Because you are exporting the function as default, you can only import it this way. https://stackoverflow.com/a/33307487

Reply:
This is more of user's preference than a rule. Use can export the component either way.

Reply:
Reverted changes